sound/soc/sof/intel/cnl.c

```diff
@@ -72,12 +73,10 @@ irqreturn_t cnl_ipc4_irq_thread(int irq, void *context)
 			spin_unlock_irq(&sdev->ipc_lock);
 		} else {
 			/* Notification received */
-			struct sof_ipc4_msg data = {{ 0 }};
+			null_data.primary = primary;
+			null_data.extension = extension;
```

Comment:
If we are renaming it, can we name it something like `rx_data` or `notification`? `null_data` does not make much sense here.

Reply:
I renamed it notification_data.